culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+expectations of them. Those acting as maintainers must be respectful,
+considerate, and judicious in the way they exercise their powers.
+
+## Lifecycle of a Review
+
+### Proposal
+The first step is for a contributor to propose a _change_.
+The contributor needs to do two things at this stage:
+- Flesh out the change (e.g. writing the code), so that there is something to review
+- Explain the change (e.g. writing a pull request description), so that reviewers can easily understand it
+
+Both steps are vital, and contributors should put real effort into explaining the change well.
+Doing so is respectful of reviewers' time, and makes it easier for others in the community to follow
+along with the project and get involved themselves.
+
+Finally, before requesting a review, the contributor should _self-review_ their change.
+It's amazing how many issues can be spotted and fixed during self-review.
+This is respectful of reviewers' time, and makes the process a lot smoother.
+
+### Review
+
+Once a change is ready and has been self-reviewed, it's ready for review. Since
+most SourceCred artifacts are tracked via repositories on GitHub, this will
+usually mean creating a pull request.
+
+Once the change is proposed, anyone in the project can add their own review.
+The person proposing the change can also request specific reviewers, if certain
+individiuals have context or knowledge thet will make their reivew particularly
+valuable.
+
+On finishing their review, the reviewer can:
+- Ask questions to better understand the change
+- Suggest changes to it
+- Approve the change
+
+Many changes go through a few rounds of reviews before being approved.
+
+### Approval and Merging
+
+During the review process, a successful change will accumulate "approvals" from
+satisfied reviewers. At some point, the change will have "enough approvals" and
+will be ready to merge. There's no hard-and-fast rule to determine when a
+change has enough approvals. Generally, simple and small changes are easy to
+approve, whereas changes that are breaking new ground, or have wide-ranging
+consequences, will require more reviews. If you're unsure as to whether a
+change is ready to merge, ask one of the maintainers.
+
+### Changes that never merge
+
+Some changes don't ever merge. A few good reasons why this may happen include:
+- Reviewers may find that the change isn't actually needed, is basically
+  unsound, or isn't aligned with the project's values or goals
+- Over the course of review, the contributor or reviewers may find a different
+  and better way to achieve the same goals
+
+# Best Practices for Reviewers
+
+## Be respectful and appreciative
+
+As a reviewer, you should be respectful and appreciative towards the person
+proposing the change. This is true especially when you quite disagree with the
+change itself. It's fine to express strong beliefs that a change is misguided,
+but you shouldn't make the contributor feel belittled.
+
```

Comment:
Yes! There are really good guides for this - even the way that you frame a suggestion is so hugely important for the experience that the contributor has. For example, instead of saying "Why did you do this it's not efficient?" it's better to say "Could you explain your thinking behind use of this data structure?" and then follow up with a logical statement like "What if we tried using [this other data structure] to create an index and then (optimize by some metric)?"

Reply:
On the same thread, there are so many cultural differences amongst different communities - some are really straight to the point and blunt (which can be interpreted as cold), and some wrap everything in flowers and ponies. We can't say there is a way to predict, but we can say to expect differences, and it's up to the individual community to largely dictate their own culture.

Reply:
Agreeing with your statements @vsoch 
Though not sure if you're suggesting a particular change. I'll defer to you if you'd like to track this / make a PR for it. :smiley: